picky-signtool/Cargo.toml

```diff
@@ -13,4 +13,4 @@ walkdir = "2.3.2"
 base64 = "0.21.0"
 encoding_rs = "0.8.32"
 lief-cwal = "0.1.0"
-picky = { version = "7.0.0-rc.3", default-features = false, features = ["wincert", "ctl", "ctl_http_fetch", "http_timestamp"] }
+picky = { version = "7.0.0-rc.6", path = "../picky", default-features = false, features = ["wincert", "ctl", "ctl_http_fetch", "http_timestamp"] }
```

Comment:
I’m not sure about this… IIRC lief-cwal is also depending on picky, but in its version `7.0.0-rc.3`, which is a problem.
I know it’s a bit weird.
Can you make sure this is still building without error when typing `cargo build` in the `picky-signtool` folder?
It’s probably best to leave `picky-signtool` untouched for now.
Somewhat related issue: https://github.com/Devolutions/picky-rs/issues/213

Reply:
Aha, I see.
`picky-signtool` builds without a problem with rc6 btw, but I'll revert this change.

Reply:
done

Reply:
Oh okay! Yeah, just to be safe and we’ll revisit that when we decide what to do about `picky-signtool` / `lief-cwal`